docroot/modules/custom/wa_orange_dam/src/Plugin/Field/FieldFormatter/DamImageFormater.php

```diff
@@ -165,6 +166,21 @@ public function viewElements(FieldItemListInterface $items, $langcode): array {
         '#attributes' => [],
         '#cache' => $cache,
       ];
+
+      $date_value = $search['APIResponse']['Items'][0]['customfield.Expiry-Date'];
```

Comment:
The expiry date is stored in a field on the media item so there's no need to be using an API call if one isn't already happening.